src/error.rs

```diff
@@ -55,10 +57,28 @@ impl fmt::Display for TiffFormatError {
             SignedIntegerExpected(ref val) => {
                 write!(fmt, "Expected signed integer, {:?} found.", val)
             }
+            InflateError(_) => write!(fmt, "Failed to decode inflate data."),
         }
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct InflateError {
+    status: TINFLStatus,
+}
+
+impl From<TINFLStatus> for TiffFormatError {
```

Comment:
The implementations of `From<_>` are part of the public interface. These two should be private inherent methods instead.

Reply:
Hmm, maybe it's not worth adding methods for then. While this is a bit wordy, it only happens in one spot:
```rust
let uncompressed = inflate::decompress_to_vec_zlib(&compressed).map_err(|status| {
    TiffError::FormatError(TiffFormatError::InflateError(InflateError::new(status)))
})?;
```
Thoughts?

Reply:
I'd still prefer a method since it doesn't capture any locals. And it takes as much visual space as the current `From<_>` impl so it shouldn't be more confusing.